TESTING.md

```diff
@@ -85,29 +84,49 @@ a few issues that need to be addressed before doing so.
 Once you have finished building and "adding" your box (with `vagrant box add`),
 you'll need to modify the `.kitchen.yml`. The only modifications you should
 need to make are replacing our box names with yours. For example, you would
-replace `apex/macos-10.13.6` with `my_high_sierra_box`. To double check the
+replace `microsoft/macos-monterey` with `my_macos_box`. To double check the
 available boxes and their names, execute `vagrant box list`. For example:
 
 ```shell
 $ vagrant box list
-apex/macos-10.12.6 (parallels, 2.0.0)
-apex/macos-10.13.6 (parallels, 1.1.0)
+microsoft/macos-big-sur     (parallels, 11.6.1)
+microsoft/macos-catalina    (parallels, 10.15.7)
+microsoft/macos-high-sierra (parallels, 10.13.6)
+microsoft/macos-mojave      (parallels, 10.14.6)
+microsoft/macos-monterey    (parallels, 12.2)
 ```
 
 Next, make sure you're in the macOS cookbook root and run `kitchen list` to view
 the available instances. It should look something like this:
 
 ```shell
 $ kitchen list
```

Comment:
This isn't a huge deal, but if we already pin the version in `kitchen.yml` would we need to put the chef version in the `platform` names? I assume its just a visibility thing but it looks like it clutters IMHO.

Reply:
This is leftover from when we tested two different versions of Chef - I'll leave it since we'll probably test 18 and 17 at the same time for a bit.